main.go

```diff
@@ -72,6 +73,7 @@ func init() {
 	utilruntime.Must(ovnv1.AddToScheme(scheme))
 	utilruntime.Must(ovsv1.AddToScheme(scheme))
 	utilruntime.Must(neutronv1.AddToScheme(scheme))
+	utilruntime.Must(ansibleeev1.AddToScheme(scheme))
```

Comment:
we don't need this.

Reply:
I'm keeping it for now because yesterday @slagle mentioned I needed to modify `main.go`

Reply:
I think we were just trying to copy the existing patterns we see in the operator. @gibizer can you explain why this wouldn't be needed? How does it differs from the other listed operators?

Reply:
I think (but I could be wrong here) that if the controller code does not try to read / write any AnsibleEE CR instance then we don't need to load the schema either.